nwbinspector/utils.py

```diff
@@ -0,0 +1,26 @@
+"""Authors: Cody Baker and Ben Dichter."""
+import numpy as np
+from collections import defaultdict
+
+global default_checks
+default_checks = {1: defaultdict(list), 2: defaultdict(list), 3: defaultdict(list)}
+
```

Comment:
If I understand this code correctly, then ``default_checks`` is used to collect all checks available, rather than defining the checks that should be executed by default. If this is the case, then I would suggest to change the name to avoid confusion. Maybe something like "available_checks"? I would also suggest to prefix the name with "_" to avoid exporting the global variable to other modules (i.e., I assume this global variable is supposed to be specific for the NWBInspector module and should never be imported by others).

Reply:
I've modified to `available_checks`, with the default behavior (and documentation) for the `inspect_nwb` being to load all available checks.

This value could be imported by other modules, as it is intended to be usable for extensions (if people write and want to add their own checks for their own extensions); certain outside uses may also want to import it to remove certain available tests (can also be done with the `skip` optional argument to `inspect_nwb`, but if there are a large number to skip every time for some particular labs conversion, maybe they'd rather just directly modify the loaded list once and just pass that).